tests/pytorch/test_loss.py

```diff
@@ -102,9 +102,13 @@ def test_register_loss_module(out_dir):
     loss_coll = trial.collection("losses")
     loss_tensor = trial.tensor("CrossEntropyLoss_output_0")
 
-    # Capture ['CrossEntropyLoss_input_0', 'CrossEntropyLoss_input_1', 'CrossEntropyLoss_output_0']
-    assert len(trial.tensor_names()) == 3
-    assert len(loss_coll.tensor_names) == 3
```

Comment:
This is a valid use case if the user (not SageMaker Metrics) wants to record the input tensor to the loss function. We should preserve this.

Reply:
agree. making the change. functional loss would also need to log inputs then